docs/README-AWS.md

```diff
@@ -0,0 +1,87 @@
+  * Click on "Browse serverless app repository"
+  * Select "Public applications" tab
+  * Click on "Show apps that create custom IAM roles or resource policies" checkbox
+  * In the search box type "elastic-serverless-forwarder" and submit
+  * Look for "elastic-serverless-forwarder" in the results and click on it
+* In the page on "Application settings" section check the "I acknowledge that this app creates custom IAM roles." checkbox
+  * Click on the "Deploy" button in the bottom right corner
+* Once on the Application page for "serverlessrepo-elastic-serverless-forwarder" loaded afterward
+  * Click on "Deployments" tab
+    * Monitor the "Deployment history" refreshing its status until the "Lambda application" "Resource type" has "Create complete" status
+* Go to "Lambda > Functions" page in the AWS console a look for the Function Name with prefix "serverlessrepo-elastic-se-ElasticServerlessForward-" and click on it
+  * Go to "Configuration" tab and select "Environment Variables"
+  * Click on "Edit" on the "Environment Variables" tab and add the environment variable `S3_CONFIG_FILE` with the value of the S3 url in the format "s3://bucket-name/config-file-name" point to the configuration file for your Elastic Forwarder for Serverless (see below)
+  * You can additionally add the following environment variables to enable Elastic APM instrumentation to your deployment of Elastic Forwarder for Serverless
+    * | Key                       | Value  |
+      |---------------------------|--------|
+      |`ELASTIC_APM_ACTIVE`       | `true` |
+      |`ELASTIC_APM_SECRET_TOKEN` | token  |
+      |`ELASTIC_APM_SERVER_URL`	  | url    |
+  * Still in the "Configuration" tab select "Triggers"
+    * You can see an already defined SQS trigger for a queue with the prefix `elastic-serverless-forwarder-continuing-queue-`. This is an internal queue and should not be modified, disabled or removed.
+    * Click on "Add trigger"
+    * From "Trigger configuration" dropdown select "SQS"
+    * In the "SQS queue" field chose the queue or insert the ARN of the queue you want to use as trigger for your Elastic Forwarder for Serverless
+      * The SQS queue you want to use as trigger must have a visibility timeout of 900 seconds, equal to the Elastic Forwarder for Serverless Lambda timeout.
+    * Click on "Add"
+
+
+### S3_CONFIG_FILE
+The Elastic Forwarder for Serverless Lambda rely on a config yaml file to be uploaded to an S3 bucket and referenced by the `S3_CONFIG_FILE` environment variable.
+
+This is the format of the config yaml file
+```yaml
+inputs:
+  - type: "sqs"
+    id: "arn:aws:sqs:%REGION%:%ACCOUNT%:%QUEUENAME%"
+    outputs:
+      - type: "elasticsearch"
+        args:
+          elasticsearch_url: "http(s)://domain.tld:port"
+          cloud_id: "cloud_id:bG9jYWxob3N0OjkyMDAkMA=="
+          username: "username"
+          password: "password"
+          api_key: "YXBpX2tleV9pZDphcGlfa2V5X3NlY3JldAo="
+          dataset: "aws.vpcflow"
```

Comment:
Add a link to the list of valid values for this field.

Reply:
I would keep the default value, helping user to choose the desired one by linking the relevant code/docs.
```suggestion
          dataset: "generic"
```

Reply:
> Add a link to the list of valid values for this field.

yes, it has to be provided.
still has to be defined how to compile it: the best would be a script to fetch from integrations repo